### PR TITLE
Awesome books: refactor to use JavaScript classes

### PR DESCRIPTION
**Pull request by ppmarq1**
Hi! We kindly ask you for a review of this project. We've completed this milestone where we were required to:

 - Use JavaScript classes instead of objects and arrays.
- Use objects and arrays, but inside classes.
- Create class methods to add and remove books.
- Maintain all of the original functionality of the application.
 - Use only plain JavaScript.
 - CSS styles to the application to make it match this wireframe:

Kind Regards,

Pedro

### CODE REVIEW

File: refactor.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+      author,
+    };
+
+    this.bkList = [...this.bkList, book];
+    localStorage.setItem('library', JSON.stringify(this.bkList));
+  }
+
+  // delete a book
+  removeBook(index) {
+    this.bkList = this.bkList.filter((b) => b.id !== index);
```
Comment (bimbolabuari) [inline refactor.js:36]: [Optional] For naming convention best practice, I suggest you give a more descriptive name to your variables and parameters.

File: refactor.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  e.preventDefault();
+  if (titleInput.value && authorInput.value) {
+    bkList.addBook(titleInput.value, authorInput.value);
+    bkList.displayBooks();
+    titleInput.value = '';
+    authorInput.value = '';
+  }
+});
+// load books list when page loads
+// window.onload = display();
```
Comment (bimbolabuari) [inline refactor.js:65]: [Optional] Let's remove all redundant code for cleaner and professional code.

**Review by bimbolabuari — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: refactor.js (hunk 1 above)
Comment (ehs9nino) [reply refactor.js:36]: @bimbolabuari  thanks we will fix this issue.

**Pull request merged**